config/kubernetes/org.yaml

```diff
@@ -1551,17 +1551,16 @@ teams:
   kube-state-metrics-admins:
     description: Admin access to the kube-state-metrics repo
     members:
-    - andyxning
```

Comment:
FYI @tariq1890 @brancz as andyxning was added to emeritus-approvers, so removing admin and write access makes sense. We can always add andyxning later on again if they decide to contribute more!

Reply:
Sounds good to me!